app/src/main/java/com/mapbox/maps/testapp/examples/style/TileJsonActivity.kt

```diff
@@ -47,6 +47,7 @@ class TileJsonActivity : AppCompatActivity() {
         rasterSource(SOURCE_ID) {
           tileSet(tileSet)
           tileSize(RASTER_TILE_SIZE_PIXELS)
+          tileRequestsDelay(5000.0) // Add tile request delay by 5seconds.
```

Comment:
Maybe we should use a new activity to demonstrate the property, or have an UI to enable/disable the tileRequestDelay. Otherwise it might look like a bug(slow loading time).

Reply:
I think instead of new activity, Ui to enable/disable the tile requests make sense. I will update that.